manifests/native.json

```diff
@@ -1039,6 +1039,94 @@
       "replacement": "Array.prototype.lastIndexOf",
       "mdnPath": "Global_Objects/Array/lastIndexOf",
       "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "abort-controller",
+      "nodeVersion": "0.10.0",
+      "replacement": "AbortController",
+      "mdnPath": "Global_Objects/AbortController",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.findlast",
+      "nodeVersion": "0.10.0",
+      "replacement": "Array.prototype.findLast",
+      "mdnPath": "Global_Objects/Array/findLast",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.findlastindex",
+      "nodeVersion": "0.10.0",
+      "replacement": "Array.prototype.findLastIndex",
+      "mdnPath": "Global_Objects/Array/findLastIndex",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.toreversed",
+      "nodeVersion": "0.10.0",
+      "replacement": "Array.prototype.toReversed",
+      "mdnPath": "Global_Objects/Array/toReversed",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.tosorted",
+      "nodeVersion": "0.10.0",
+      "replacement": "Array.prototype.toSorted",
+      "mdnPath": "Global_Objects/Array/toSorted",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.tospliced",
+      "nodeVersion": "0.10.0",
+      "replacement": "Array.prototype.toSpliced",
+      "mdnPath": "Global_Objects/Array/toSpliced",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.with",
+      "nodeVersion": "0.10.0",
+      "replacement": "Array.prototype.with",
+      "mdnPath": "Global_Objects/Array/with",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "string.prototype.repeat",
+      "nodeVersion": "0.10.0",
+      "replacement": "String.prototype.repeat",
+      "mdnPath": "Global_Objects/String/repeat",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "is-builtin-module",
+      "nodeVersion": "0.10.0",
+      "replacement": "Use `isBuiltin` from 'node:module'",
```

Comment:
This one and `md5` are native, but don't have an mdnPath, how do you want to handle those? Just leave it empty?

Reply:
it should be a `simple` entry, or a `documented` one if we need extra wording around migration

in this case, we already have it here:

https://github.com/es-tooling/module-replacements/blob/7ab526fdcc36e2be4a09d2b28e2e0c16aa606b0d/manifests/preferred.json#L51-L62

Reply:
for the `md5` one, we should probably just add it to the preferred manifest and give it a doc

what do you think?

Reply:
moved md5 to preferred, removed is-builtin-module; it was already there